tests/system/lab/test

```diff
@@ -863,14 +862,26 @@ event_toggled_sent_clear
 
 echo "======================= TOGGLE EVENT NTFSN and NTFCL TEST COMPLETE ======================="
 
+if [[  ${VERIFY_EGRESS_TO_PI} == 1 ]]
+  then
+     setup_north_pi_egress
+     curl -X PUT  http://localhost:8081/fledge/schedule/disable -d '{"schedule_name": "PI Server"}'
+     # Wait for North Service to shutdowm properly.
+     sleep 60
```

Comment:
We can add poll way for `/service` and check the `status = shutdown` status for the disabled service